lib/queries.js

```diff
@@ -119,68 +183,431 @@ export async function updateOvoNumberAndUri(ovoStructuredIdUri, wegwijsOvo) {
+                                              contactPointUuidId)} `
+  if(kboOrg.email){
+    updateStr += `; <http://schema.org/email> ${sparqlEscapeString(
+      kboOrg.email
+    )}`
+  }
+  if(kboOrg.phone){
+    updateStr += `; <http://schema.org/telephone> ${sparqlEscapeString(
+      kboOrg.phone
+    )}`
+  }
+  if(kboOrg.website){
+    updateStr += `; <http://xmlns.com/foaf/0.1/page> ${sparqlEscapeString(
+      kboOrg.website
+    )}`
+  }  
+  updateStr += `
+    ; <http://schema.org/contactType> ${sparqlEscapeString("Kbocontact")} ;
+    <http://www.w3.org/ns/locn#address> ${sparqlEscapeUri(adressUri)}.`                                         
+
+
+
+  updateStr +=`    ${sparqlEscapeUri(
+        newKboIdUri
+      )} a <http://mu.semte.ch/vocabularies/ext/KboEenheid> ;
+                                        <http://mu.semte.ch/vocabularies/core/uuid> ${sparqlEscapeString(
+                                          newKboOrgIdUuId
+                                        )}`
+
+  if(kboOrg.rechtsvorm){
+    updateStr += `; <http://mu.semte.ch/vocabularies/ext/rechtsvorm> ${sparqlEscapeString(
+      kboOrg.rechtsvorm
+    )}` 
+  }                                      
+  if(kboOrg.shortName){
+    updateStr += `; <http://mu.semte.ch/vocabularies/ext/shortName> ${sparqlEscapeString(
+      kboOrg.shortName
+    )}` 
+  }                                      
+  if(kboOrg.startDate){
+    updateStr += `; <http://mu.semte.ch/vocabularies/ext/startDate> ${sparqlEscapeString(
+      kboOrg.startDate
+    )}` 
+  }                                      
+  if(kboOrg.formalName){
+    updateStr += `; <http://www.w3.org/2004/02/skos/core#altLabel> ${sparqlEscapeString(
+      kboOrg.formalName
+    )}` 
+  }                                      
+  if(kboOrg.organisationName){
+    updateStr += `;  <http://www.w3.org/2004/02/skos/core#prefLabel> ${sparqlEscapeString(
+      kboOrg.organisationName
+    )}` 
+  }                                      
+  if(kboOrg.changeTime){
+    updateStr += `; <https://data.lblod.info/vocabularies/generiek/geplandeEindDatum> ${sparqlEscapeString(
+      kboOrg.changeTime
+    )}` 
+  }                                      
+  if(kboId){
+    updateStr += `; <http://www.w3.org/ns/adms#identifier> ${sparqlEscapeUri(
+      kboId
+    )}` 
+  }                                      
+  if(kboOrg.activeState){
+    updateStr += `; <http://www.w3.org/ns/regorg#orgStatus> ${sparqlEscapeUri(
+      kboOrg.activeState
+    )}` 
+  }                                      
+  updateStr += `; <http://schema.org/contactPoint> ${sparqlEscapeUri(
+          contactPointUri
+        )} .    
+      }
+  }WHERE{
+    VALUES ?g {
+      <http://mu.semte.ch/graphs/administrative-unit>
+    }
+  }   
+  `;
+
+  await update(updateStr);
+  return newKboIdUri;
+}
+
+export async function updateKboOrg(newKboInfo, identifiers) {
```

Comment:
**Suggestion:** I would extract the 3 queries performed in this function to their own separate functions, and call these functions from here. (You could make the 3 new functions private to ensure they are not called separately.) Between all the SPARQL stuff it is easy to miss an `await update(updateStr)`, making it harder to understand what queries will actually be executed.

Reply:
extracted to 3 functions